### ISSUE TITLE
README.md not up to date with library

### ISSUE THREAD

**Issue by samuelhaysom** [alisterburt/starfile#28]
Currently the README.md file says under usage:

Opening STAR files containing multiple dataframes blocks will return a dict of DataFrame objects.
If you would like to always return a dict, you can use the always_dict=True keyword argument

However when I have used starfile.read to import star files I instead have got a list of Dataframes rather than a Dict. Additionally, the always_dict keyword argument doesn't work and when I look at the docstring for starfile.read it is not mentioned.

I am using v 0.111 installed from PyPi using pip

**Comment by alisterburt** [comment]
Hi @samuelhaysom - thanks for the message

You're using an old version of starfile, probably because you're using an old version of Python not supported by the latest version of starfile

https://pypi.org/project/starfile/#history

I'm vaguely keeping in line with timescales provided in [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html)

Let me know if you still have problems in an environment with python 3.8+

**Comment by samuelhaysom** [comment]
Hi @alisterburt,

Ah I see, apologies I should have checked my version number before posting. I was running python 3.7.1 in the environment I used to install the package.

I've now tested it in python 3.10.0 and it all works as it should.

While this was my error, maybe its worth adding a line in the README about supported python versions? Currently the python versions badge displayed by github at the top of the README indicates the package should work on 3.7.1 (which I does I guess but not in the way indicated in the README.md)
